### PR TITLE
WIP-165 update and check APIs for all examples

### LINKED ISSUE #165: update and check APIs for all examples
**Is your feature request related to a problem? Please describe.**
As we developed many new features and APIs recently.
Need to update all the examples, add more comments to introduce and double-check again.

### PR DESCRIPTION
**Pull request by Nic-Ma**
Fixes #165 .

### Description
This PR implemented:
1. Updated all the APIs in all the examples to the latest version.
2. Fixed many errors in examples.
3. Changed UNet output to only logits.

### Status
**Work in progress**

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or new feature that would cause existing functionality to change)
- [ ] New tests added to cover the changes
- [x] Docstrings/Documentation updated

### CODE REVIEW

**Comment by Nic-Ma** [comment]
merge it to 3D classification example branch directly.
And will continue to work on just 1 branch.
